ds3-sdk/src/test/java/com/spectralogic/ds3client/serializer/XmlOutput_Test.java

```diff
@@ -95,4 +96,18 @@ public void toXmlWithPriority() throws XmlProcessingException {
 
         assertThat(result, is(expectedString));
     }
+
+    @Test(expected = java.io.IOException.class)
+    public void fromXmlWithInvalidElementThrowsExceptionInDevBuild() throws IOException {
```

Comment:
Won't this test fail on a production build because the expected exception was not thrown?